docs/source/install.rst

```diff
@@ -1,6 +1,6 @@
 Installing and Configuring Quipucords
 =====================================
-Quipucords is delivered in two parts, a command line tool as an RPM package and a server as a container image. The following instructions describe how to install the parts of Quipucords by using a scripted installation or by using step-by-step commands.
+Quipucords is delivered in three parts, a command line tool as an RPM package and the server and database as a container image. The following instructions describe how to install the parts of Quipucords by using a scripted installation or by using step-by-step commands.
```

Comment:
Change the first sentence to: `Quipucords is delivered in two parts, a command line tool as an RPM package and a server as container images.`